src/components/ebay-carousel/style.less

```diff
@@ -1,115 +1,135 @@
-.carousel {
-    @transition-time: 0.25s;
+@ebay-carousel-transition-time: 0.25s;
+@ebay-carousel-transition-function: ease-out;
+@ebay-carousel-control-enabled-opacity: 0.92;
+@ebay-carousel-control-disabled-opacity: 0.3;
 
+.carousel {
     &__container {
         white-space: nowrap;
         overflow-y: hidden;
         overflow-x: hidden;
-        -webkit-overflow-scrolling: touch;
         padding: 16px 0;
         position: relative;
         width: 100%;
     }
 
     &__list {
-        box-sizing: border-box;
-        position: relative;
+        transition: transform @ebay-carousel-transition-time @ebay-carousel-transition-function;
         transform: translateX(0);
-        transition: transform @transition-time;
-
+        position: relative;
+        box-sizing: border-box;
         width: 100%;
         margin: 0;
         padding: 0;
 
         > li {
             list-style: none;
-            margin-right: 8px;
+            margin-right: 16px;
+
+            &:last-of-type {
+                margin-right: 0;
+            }
         }
     }
 
-    &--continuous &__list {
-        display: inline-block;
+    &__control {
+        transition: opacity @ebay-carousel-transition-time @ebay-carousel-transition-function;
+        opacity: 0;
+        pointer-events: none;
+        font-size: 18px;
+        background-color: @color-core-white;
+        padding: 0; // override browser default
+        height: 72px;
+        width: 28px;
+        z-index: 1;
+        border: 1px solid @color-core-gray-jet;
+        cursor: pointer;
+        position: absolute;
+        top: 50%;
+        transform: translateY(-50%);
 
-        > li {
-            display: inline-block;
+        &--prev {
+            left: 0;
         }
-    }
 
-    &--discrete &__list {
-        display: flex;
+        &--next {
+            right: 0;
+        }
 
-        > li {
-            flex: 0 0 100%;
+        &.icon:before {
+            color: @color-core-gray-jet;
```

Comment:
This appears to be a DS4 color? I think it needs to change to a DS6 color, possibly `@ds6-color-g206-gray`.

http://ebay.github.io/skin/ds6/#less